src/components/ApplicantDetail/container.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FunctionComponent } from "react";
+import { getTranslations } from "$queries";
+import { ApplicantDetail } from "./component";
+import { useQuery } from "@apollo/react-hooks";
+
+const ApplicantDetailContainer: FunctionComponent = () => {
+  const { data } = useQuery(getTranslations, {
+      variables: {
+        paths: [
+          "applicant.padron",
+          "applicant.capabilities",
+          "applicant.careers",
+          "applicant.credits"
+        ]
+      }
+    }
+  );
+  const [ padron, capabilities, careers, credits ] = data ? data.getTranslations : ["", "", "", ""];
+
+  return (
+    <ApplicantDetail
+      name={"Sebastian"}
+      surname={"Blanco"}
+      padron={98539}
+      description={
+        "Me considero una persona graciosa, con talentos de buen humor. Tengo promedio " +
+        "8 en la facultad y por eso me considero una personasabia con aires de " +
+        "grandeza."
+      }
+      careers={
+        [
+          {
+            name: "Ingeniería Informática",
+            credits: 200
+          },
+          {
+            name: "Ingeniería Civil",
+            credits: 20
+          }
+        ]
+      }
+      capabilities={ ["Python", "Node", "css", "Auth security"] }
+      translations={
```

Comment:
Aca no veo que sea necesaria la desestructuracion en el Array para estructurarlo aca y para luego usarlo como "translations.<property>". Creo que directamente me traeria el array sin desestructurarlo y luego aca haria algo como: 
```ts
const translations = data ? data.getTranslations : ["", "", "", ""];
.
.
.
translations= {...translations}
```

Reply:
Aunq ya estoy viendo que esta forma de manejar el loading de la query no estaria escalando

Reply:
Eso igual no se puede hacer, porque las translation que recibo son un array y to le paso un hash al componente. 
esto no me lo permite hacer: ```translations= {...translations}```